### PR TITLE
Pin lhotse version to 1.19.2

### PR DESCRIPTION
**Pull request by pzelasko**
# What does this PR do ?

Pin lhotse version to 1.19.2. Prevents breaking changes.

**Collection**: All speech collections

# Changelog 
- Pin lhotse version to 1.19.2

# Usage
* You can potentially add a usage example below

```python
# Add a code snippet demonstrating how to use this 
```

# Jenkins CI
To run Jenkins, a NeMo User with write access must comment `jenkins` on the PR.

# Before your PR is "Ready for review"
**Pre checks**:
- [ ] Make sure you read and followed [Contributor guidelines](https://github.com/NVIDIA/NeMo/blob/main/CONTRIBUTING.md)
- [ ] Did you write any new necessary tests?
- [ ] Did you add or update any necessary documentation?
- [ ] Does the PR affect components that are optional to install? (Ex: Numba, Pynini, Apex etc)
  - [ ] Reviewer: Does the PR have correct import guards for all optional libraries?
  
**PR Type**:
- [ ] New Feature
- [ ] Bugfix
- [ ] Documentation

If you haven't finished some of the above items you can still open "Draft" PR.

## Who can review?

Anyone in the community is free to review the PR once the checks have passed. 
[Contributor guidelines](https://github.com/NVIDIA/NeMo/blob/main/CONTRIBUTING.md) contains specific people who can review PRs to various areas.

# Additional Information
* Related to # (issue)

### CODE REVIEW

**Comment by pzelasko** [comment]
jenkins

File: requirements/requirements_asr.txt [hunk 1]
```diff
@@ -5,7 +5,7 @@ ipywidgets
 jiwer
 kaldi-python-io
 kaldiio
-lhotse>=1.19.2
+lhotse==1.19.2
```
Comment (titu1994) [inline requirements/requirements_asr.txt:8]: Is it necessary to pin main ? What about r1.23 - that needs the pin too

**Review by titu1994 — APPROVED** [review]

File: requirements/requirements_asr.txt (hunk 1 above)
Comment (pzelasko) [reply requirements/requirements_asr.txt:8]: Yes it is necessary

**Comment by titu1994** [comment]
Merging as is since its urgent. @pzelasko  lets pin r1.23 as well asap (send me PR)

**Pull request merged**